CMakeLists.txt

```diff
@@ -9,14 +9,15 @@ option(FAASM_SELF_TRACING "Turn on system tracing using the logger" OFF)
 option(FAASM_PERF_PROFILING "Turn on profiling features as described in debugging.md" OFF)
 
 # WAMR and interpreter
-option(WAMR_INTERPRETER_MODE "Turns interpreter on / off (vs. AoT)" OFF)
+option(FAASM_WAMR_AOT_MODE "Turns on AoT mode in WAMR" ON)
 
 # SGX functionality
 # These options customise the SGX features _provided_ SGX is found
 option(FAASM_SGX_SIM_MODE "Turn on SGX sim mode" ON)
 option(FAASM_SGX_ATTESTATION "Turn on attestation" OFF)
 option(FAASM_SGX_XRA "Turn on support for XRA" OFF)
-option(FAASM_SGX_WAMR_AOT_MODE "Turn on AoT mode in SGX" ON)
+option(FAASM_SGX_WAMR_AOT_MODE "Turn on AoT mode in SGX" OFF)
+set (FAASM_SGX_WAMR_AOT_MODE OFF)
```

Comment:
Until we decide how we choose between test modes, not having this `set` means that you must manually change the option, which is cached in `CMakeCache.txt`. I.e. re-build from scratch.